sh/utils/set_aliases.sh

```diff
@@ -77,6 +77,13 @@ alias stests-set-node='_exec_cmd $STESTS_PATH_CLI/cache/set_node.py'
 alias stests-set-node-bonding-key='_exec_cmd $STESTS_PATH_CLI/cache/set_node_bonding_key.py'
 alias stests-set-node-status='_exec_cmd $STESTS_PATH_CLI/cache/set_node_status.py'
 
+# ###############################################################
+# ALIASES: cache - vw
+# ###############################################################
+
+alias stests-vw-block-info='_exec_cmd $STESTS_PATH_CLI/cache/view_block_info.py'
+alias stests-vw-deploy-info='_exec_cmd $STESTS_PATH_CLI/cache/view_deploy_info.py'
```

Comment:
I would personally prefer if those were called `stests-view-block-info` and `stests-view-deploy-info`. The names are long, anyway, so I always use the shell tab name expansion feature in the shell interactive mode to get the full name. I would rather see a list of non-ambigous, non-car related names after typing `stests` and hitting tab.

Reply:
Implemented your suggestion.